doc/troubleshooting/index.rst

```diff
@@ -0,0 +1,27 @@
+FAQs
+====
+In consequence of various differences of computers or systems, problems may occur. Some common circumstances are listed as follows. 
+In addition, some frequently asked questions about parameters setting are listed as follows.
+If other unexpected problems occur, you're welcome to contact us for help.
+
+.. _trouble:
+
+.. toctree::
```

Comment:
For this directory to handle possible changes in the future, may we consider this:
```python
.. toctree::
    :maxdepth: 1
    :caption: trouble shooting
    :glob:

    ./*
```

Reply:
The sections in "doc/troubleshooting" will be ordered alphebatically. And the subsections will be lost. About this, I think "troubleshooting" is a problem pool for people to mine in, so subsections may not be necessary.

Reply:
This turned out to work out fine.